src/arrays.jl

```diff
@@ -85,28 +83,29 @@ function Base.cat(us::UniFinArr{S,V,R,P,N}...;
     isempty(us) && return []
```

Comment:
Seems some comments here still refer to the old code.

Reply:
good catch. i just removed the comment to save confusion